src/manifolds/Decorator.jl

```diff
@@ -173,7 +185,7 @@ end
     M::AbstractDecoratorManifold, p, σ
 )
 
-@decorator_transparent_function project_point!(M::AbstractDecoratorManifold, q, p)
+@decorator_transparent_function project_point!(M::AbstractDecoratorManifold, p)
```

Comment:
I think we agreed to make `project_point!` a three-argument function (it can mutate a different point than the projected one). It's this way in ManifoldsBase.jl but I can see that not all places in Manifolds.jl conform.

Reply:
Ah, ok.